src/finch/interface/lazy.py

```diff
@@ -1660,3 +1684,29 @@ def std(
     x = defer(x)
     d = var(x, axis=axis, correction=correction, keepdims=keepdims)
     return pow(d, 0.5)
+
+
+def argmin(x, /, *, axis: int | None = None, keepdims: bool = False, init=None):
+    """
+    Returns the indices of the minimum values in input array ``x`` along given axis.
+    """
+    if isinstance(axis, tuple):
+        raise ValueError("Type of axis should is only allowed to be int or None.")
+
+    indices = np.indices(x.shape)
+    paired_tensor = pairwise_indices(x, indices)
```

Comment:
just call `elementwise(Pair, values, indices)`. no need for a complex custom function.